apps/storybook/src/Slider.stories.tsx

```diff
@@ -20,19 +20,36 @@ export default {
   args: {
     thumbMode: 'inhibit-crossing',
     trackDisplayMode: 'auto',
+    orientation: 'horizontal',
   },
 } as Meta<SliderProps>;
 
+const sliderWrapper = (element: ReactNode, args: object) => {
+  return (
+    // TODO: Is there a way to avoid the hardcoded height of 400px? height of 100% does not work
+    <div
+      className='slider-wrapper'
+      style={{
```

Comment:
On line 27, I had to change the type of args from `any` to `object`. This was because of the below error when `yarn lint` was run by GitHub actions. Why am I receiving this error? Why is any wrong here?

![image](https://user-images.githubusercontent.com/45748283/180875596-fed85122-349e-4059-9a9f-ef912d668725.png)

Reply:
Change the type here from `object` to `SliderProps` since you're only passing in `SliderProps` with this function. It should also correct the error when trying to access `args.orientation`

Reply:
**[DONE]** Ohh, right, makes sense, thanks

Reply:
I don't see the change, but it could be an issue on my end

Reply:
Yes, sorry about that, I forgot to push it 😁. Please let me know if it is still not visible